### ISSUE TITLE
Accessibility issue: <svg> elements with an img role must have an alternative text on /metrics/

### ISSUE THREAD

**Issue by weboverhauls** [weboverhauls/opensource.guide#56]
## What
An accessibility scan flagged the element `<svg height="18" class="octicon octicon-triangle-down ml-2 fill-blue v-align-middle icon-flip" viewBox="0 0 12 16" version="1.1" width="13" role="img">
            <path fill-rule="evenodd" d="M0 5l6 6 6-6z"></path>
          </svg>` because <svg> elements with an img role must have alternative text. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/svg-img-alt?application=playwright.

To fix this, ensure <svg> elements with an img, graphics-document or graphics-symbol role have accessible text.

Specifically:

Fix any of the following:
-   Element has no child that is a title
-   aria-label attribute does not exist or is empty
-   aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
-   Element has no title attribute

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
